src/ConferenceItem.jsx

```diff
@@ -31,6 +32,7 @@ const ConferenceItem = ({ conference }) => {
         <p>{getDatestring(date)}</p>
         <p>{getTimestring(date)}</p>
         <p>{city}</p>
+        <Button> Test </Button>
```

Comment:
I thought I left a comment here but I can't see it... Recommend adding padding to your button instead of doing it with spaces either side of the text

Reply:
Good point, although I think this was just how we wrote this rather than trying to add padding.